### DISCUSSION TITLE
Publication status shouldn't show "Ongoing" when it is unknown

### DISCUSSION THREAD

**Discussion by nalsai** [Kareadita/Kavita#2993] [Ideas]
### Idea Description

I have manga that do not have any metadata concerning the publication status attached.
I think that it is weird that Kavita always shows "Ongoing" even when the manga has ended, and I have it completely, but Kavita just has no way of knowing.

It should rather show "Unknown" or just be hidden.

### Idea Category

User Experience

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Ongoing is the default state. Once you fill out the appropriate metadata, it can then be ended or completed. 

I'm in disagreement with having unknown be the default or hiding when unknown.
